<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1294</width>
    <height>655</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Graphical Single Cell Viewer</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <widget class="QToolButton" name="zoom_in">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="zoom_out">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="save_viewport">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="copy_viewport">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QSplitter" name="right_splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QWidget" name="verticalLayoutWidget">
       <layout class="QVBoxLayout" name="opengl_view_layout"/>
      </widget>
      <widget class="QTabWidget" name="right_panels">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Ignored" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="currentIndex">
        <number>-1</number>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1294</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionImport"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionView_help"/>
    <addaction name="actionReport_bug"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Visualize</string>
    </property>
    <widget class="QMenu" name="menuSave_viewport">
     <property name="title">
      <string>Save viewport</string>
     </property>
     <addaction name="actionSave_as_File"/>
     <addaction name="actionCopy_to_clipboard"/>
    </widget>
    <addaction name="actionCell_color"/>
    <addaction name="actionColor_scheme"/>
    <addaction name="actionCamera_settings"/>
    <addaction name="separator"/>
    <addaction name="menuSave_viewport"/>
   </widget>
   <widget class="QMenu" name="menuSettings_2">
    <property name="title">
     <string>Settings</string>
    </property>
    <addaction name="actionSettings"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuSettings"/>
   <addaction name="menuSettings_2"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionReport_bug">
   <property name="text">
    <string>Report bug</string>
   </property>
  </action>
  <action name="actionCell_color">
   <property name="text">
    <string>Cell color</string>
   </property>
  </action>
  <action name="actionColor_scheme">
   <property name="text">
    <string>Color scheme</string>
   </property>
  </action>
  <action name="actionCamera_settings">
   <property name="text">
    <string>Camera settings</string>
   </property>
  </action>
  <action name="actionImport">
   <property name="text">
    <string>Import</string>
   </property>
  </action>
  <action name="actionSave_as_File">
   <property name="text">
    <string>Save as image</string>
   </property>
  </action>
  <action name="actionCopy_to_clipboard">
   <property name="text">
    <string>Copy to clipboard</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="text">
    <string>Settings...</string>
   </property>
  </action>
  <action name="actionView_help">
   <property name="text">
    <string>View help</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
